test/syscall_test/Makefile

```diff
@@ -32,7 +32,18 @@ TESTS ?= \
 	pty_test \
 	read_test \
 	readv_test \
+	rename_test \
+	sendfile_test \
+	stat_test \
 	stat_times_test \
+	statfs_test \
+	symlink_test \
+	sync_test \
+	timers_test \
+	truncate_test \
+	uidgid_test \
+	unlink_test \
+	vdso_clock_gettime_test \
 	write_test \
 	utimes_test \
```

Comment:
Please move the `utimes_test` after `unlink_test`

Reply:
get it